js/detail.js

```diff
@@ -0,0 +1,62 @@
+// DATA ==================================================================
+let userData = [];
+let products = JSON.parse(localStorage.getItem("products"));
+
+let indexOfProduct = JSON.parse(localStorage.getItem("productIndex"));
+
+let product = products[indexOfProduct];
+
+
+let rating = document.querySelector("#rating");
+// Display details products -----------------------------------------------
+document.querySelector("#img").src = product.image;
+document.querySelector("#name").textContent = product.name;
+document.querySelector("#description").textContent = product.description;
+document.querySelector("#comment").textContent = product.comment;
+document.querySelector("#type").textContent = product.type;
+document.querySelector("#price").textContent = product.price + product.currency;
+
+// FUNCTIONS ==============================================================
+// Save data user ---------------------------------------------------------
+function saveData() {
+    localStorage.setItem("userData", JSON.stringify(userData));
+}
+
+// Load data user ---------------------------------------------------------
+function loadData() {
+    let localDataUser = JSON.parse(localStorage.getItem("userData"));
+    if (localDataUser != null) {
+        userData = localDataUser;
+    }
+}
+
+// Get user add cart ------------------------------------------------------
+function getProduct() {
+    userData.push(product);
+    saveData();
+    my_cart.textContent = "MY CART   + " + userData.length;
```

Comment:
Use interpolation instead of concatenation: 
```javascript
    my_cart.textContent = "MY CART   + " + userData.length;
```
=> 

```javascript
    my_cart.textContent = `MY CART ${userData.length)`;
```